backend/lib-shared-types/src/dto/site_api/get_site_dto.rs

```diff
@@ -19,6 +19,30 @@ pub struct GetSiteDto {
     pub disabled: Option<bool>,
 }
 
+#[derive(Deserialize)]
+pub struct GetSiteQuery {
+    pub update_key: Option<String>,
```

Comment:
Frontend only needs to take the `updated_at` received in the response, such as `2023-11-08T02:39:36Z` and use it as the update key